static/app/views/issueDetails/traceTimeline/traceTimelineTooltip.tsx

```diff
@@ -52,13 +56,21 @@ export function TraceTimelineTooltip({event, timelineEvents}: TraceTimelineToolt
           <Link
             to={generateTraceTarget(event, organization, location)}
             onClick={() => {
-              trackAnalytics(
-                'issue_details.issue_tab.trace_timeline_more_events_clicked',
-                {
-                  organization,
-                  num_hidden: filteredTimelineEvents.length - 3,
-                }
-              );
+              if (surface.includes('issue_details')) {
```

Comment:
Tbh this depends on how issues wants to setup surfaces for their pages and components. Assuming we make a surface for every top-level view, it could be `issues.details` here. Or more explicit: `issues.issue_details`. I have both styles in the code right now, both `feedback.details` and `issues.issue_details`. We should probably pick one.